blueprints/component-test/index.js

```diff
@@ -22,13 +39,41 @@ module.exports = {
   locals: function(options) {
     var dasherizedModuleName = stringUtil.dasherize(options.entity.name);
     var componentPathName = dasherizedModuleName;
-    if(options.pod && options.path !== 'components' && options.path !== '') {
+    var testTypeDefinition = "integration: true";
+    var friendlyTestDescription = testInfo.description(options.entity.name, "Integration", "Component");
+    var testContent = "assert.expect(1);" + EOL + EOL +
+      "  // Set any properties with this.set('myProperty', 'value');" + EOL +
+      "  // Handle any actions with this.on('myAction', function(val) { ... });" + EOL + EOL +
+      "  // Provide a template (string or precompiled) for this.render()" + EOL +
+      "  this.render();" + EOL + EOL +
```

Comment:
Something needs to be passed to `render`, as otherwise an [error is thrown](http://git.io/vIPAi). The problem is now that the `ember-template-compiler.js` needs to be present ... :cry:

I plan to open a PR for including `htmlbars-inline-precompile` as default, so this line will eventually be changed to:

``` js
this.render(hbs`
  {{<%= dasherizedModuleName %>}}
`);
```

Not sure what to do now? Sadly the current version of the integration test will throw an error, but maybe that's ok and this should be fixed before a new release is cut..